packages/ymax-planner/src/engine.ts

```diff
@@ -29,7 +35,8 @@ type CosmosEvent = {
   attributes?: Array<{ key: string; value: string }>;
 };
 
-const VSTORAGE_PATH_PREFIX = 'published.ymax0.portfolios';
+const PORTFOLIOS_PATH_PREFIX = 'published.ymax0.portfolios';
+const PENDING_TXS_PATH_PREFIX = 'published.ymax0.PendingTxs';
```

Comment:
Comprehensibility suggestion to avoid misinterpretation of "TXS" as an initialism:
```suggestion
const TX_SUBSCRIPTIONS_PATH_PREFIX = 'published.ymax0.PendingTxs';
```